Editor/EditorCore/DrawShapeTool.cs

```diff
@@ -406,13 +415,29 @@ void OnOverlayGUI(UObject overlayTarget, SceneView view)
             }
         }
 
+        void ResetPrefs()
+        {
+            var type = EditorShapeUtility.availableShapeTypes[s_ActiveShapeIndex];
+            if(currentShapeInOverlay == m_LastShapeCreated)
+                m_LastShapeCreated = null;
+
+            UndoUtility.RegisterCompleteObjectUndo(currentShapeInOverlay, "Change Shape");
+            currentShapeInOverlay.SetShape(EditorShapeUtility.CreateShape(type), currentShapeInOverlay.pivotLocation);
+            SetBounds(currentShapeInOverlay.size);
```

Comment:
I don't think the active shape is a setting that should respect the "reset all" action. If it's not already the case, it should be that these operational types of settings (ex, active edit mode or current shape) should be stored irrespective of the project or user level preferences.

Reply:
Indeed, the active shape does not need to respect the resetall function. This can be removed.
However the shape settings still should on their side